docs/sources/project/windows-client-test-server.md

```diff
@@ -0,0 +1,139 @@
+page_title: Set up a Windows server for client testing
+page_description: How to set up a server to test Docker Windows client
+page_keywords: development, inception, container, image Dockerfile, dependencies, Go, artifacts, windows
+
+
+# Windows Client test server
```

Comment:
how about `Building and testing the Windows Docker client`? otherwise, someone looking to find our how to build Docker on windows might not read this one.